cmd/par/main.go

```diff
@@ -1,6 +1,9 @@
 package main
 
 import (
+	"C"
```

Comment:
Sure! I'm however not sure how symbol visibility works in Go: do I import lib/libgopar into main then? Shoudl `par()` also go there, in case you use it as a Go library?

Reply:
(Since I currently use `par2cmdline` as a library, this was the easiest way to test it! Once this is all figured out, and I have some more time, my plan is to port par2deep to Go! Should help move parity files into the UTF8 era. And a great cross-compilation story of course.)

Reply:
main shouldn't import anything new, but it should export a new `Par()` function. There'd be a new package `libgopar` which would then import `cmd/par` and use `Par()`. Then you should be able to build the C library from `libgopar`, instead of from `cmd/par`, I think. Let me know if that works.